apps/passwd.c

```diff
@@ -43,13 +50,13 @@ static unsigned const char cov_2char[64] = {
 static int do_passwd(int passed_salt, char **salt_p, char **salt_malloc_p,
                      char *passwd, BIO *out, int quiet, int table,
                      int reverse, size_t pw_maxlen, int usecrypt, int use1,
-                     int useapr1);
+                     int useapr1, int use5, int use6);
```

Comment:
i'd really prefer a single 'mode' variable, rather than adding a new flag param, since only one flag can be set at a time ...